index.js

```diff
@@ -0,0 +1,79 @@
+const D = require('date-fp');
+const S = require('sanctuary');
+const $ = require('sanctuary-def');
+
+const a = $.TypeVariable('a');
+const b = $.TypeVariable('b');
+
+const $TimeUnit = $.EnumType([
+  'milliseconds',
+  'seconds',
+  'minutes',
+  'hours',
+  'days']);
+
+const $TimeUnitFull = $.EnumType([
+  'milliseconds',
+  'seconds',
+  'minutes',
+  'hours',
+  'days',
+  'months',
+  'years']);
+
+const $TimeUnitGet = $.EnumType([
+  'milliseconds',
+  'seconds',
+  'minutes',
+  'hours',
+  'date',
+  'month',
+  'year']);
+
+const $Either = $.BinaryType(
+  'sanctuary/Either',
```

Comment:
Strictly speaking this should live in your package's namespace, since your package is defining the type.

I'd welcome a pull request to have Sanctuary export this type. :)